doc/memory.md

```diff
@@ -0,0 +1,17 @@
+# Memory
+
+ROHD HCL provides a generic `abstract` [`Memory`](https://intel.github.io/rohd-hcl/rohd_hcl/Memory-class.html) class which accepts a dynamic number of `writePorts` and `readPorts`, where each port is of type [`DataPortInterface`](https://intel.github.io/rohd-hcl/rohd_hcl/DataPortInterface-class.html).  A `DataPortInterface` is a simple interface with `en` and `addr` as `control` signals and `data` signal(s).  In a write interface, all signals are in the same direction.  In a read interface, the `control` signals are in the opposite direction of the `data` signal(s).
+
+## Strobes
+
+A sub-class of `DataPortInterface` is the[`StrobeDataPortInterface`](https://intel.github.io/rohd-hcl/rohd_hcl/StrobeDataPortInterface-class.html), which adds `strobe` to the `data` group of signals.  The `strobe` signal is a byte-enable signal, where each bit of `strobe` controls one byte of `data`.
```

Comment:
It would be good to use the best terminology.  Do you think `byteMask` is a better, more standard name than `strobe`, `byteEnable`, or something else?  Whichever we pick, it might be good to write in the docs the different names if there are multiple sufficiently popular options.

Reply:
byte-enable is more popular

Reply:
Given #15, should we stick with "strobe" as the name since "byte enable" and "byte mask" are granularity-specific names?  Any other suggestions on the name?

Reply:
Or maybe "masked" or "enabled" is better?

Reply:
I think "mask" is better because it's confusing for an interface to have an enable for the interface, and another signal like "byte enable" but without the word "byte" on it.